preview-screenshots/src/test/java/io/github/droidkaigi/confsched2022/PreviewScreenshotTests.kt

```diff
@@ -0,0 +1,79 @@
+package io.github.droidkaigi.confsched2022
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.unit.Density
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.androidHome
+import app.cash.paparazzi.detectEnvironment
+import com.airbnb.android.showkase.models.Showkase
+import com.airbnb.android.showkase.models.ShowkaseBrowserComponent
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameter.TestParameterValuesProvider
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
```

Comment:
Just for confirmation 👀 , is Paparazzi possible to working with JUnit5? 🤔  if possible, other engineer can use modern toolchain...! ✨

Reply:
I didn't check this one.
I would like to get it working first.:pray: